scripts/monitoring/monitor_ecs_metrics.py

```diff
@@ -0,0 +1,226 @@
+import os
+import pytz
+from evalai_interface import EvalAI_Interface
+from datetime import datetime, timedelta
+
+from django.conf import settings
+
+utc = pytz.UTC
+auth_token = os.environ.get(
+    "AUTH_TOKEN",
+)
+evalai_endpoint = os.environ.get("API_HOST_URL")
+authorization_header = {
+    "Authorization": "Bearer {}".format(os.environ.get("AUTH_TOKEN"))
+}
+
+DJANGO_SETTINGS_MODULE = os.environ.get("DJANGO_SETTINGS_MODULE")
+ENV = DJANGO_SETTINGS_MODULE.split(".")[-1]
+EVALAI_DNS = os.environ.get("SERVICE_DNS")
+
+aws_keys = {
+    "AWS_ACCOUNT_ID": os.environ.get("AWS_ACCOUNT_ID", "x"),
+    "AWS_ACCESS_KEY_ID": os.environ.get("AWS_ACCESS_KEY_ID", "x"),
+    "AWS_SECRET_ACCESS_KEY": os.environ.get("AWS_SECRET_ACCESS_KEY", "x"),
+    "AWS_REGION": os.environ.get("AWS_DEFAULT_REGION", "us-east-1"),
+    "AWS_STORAGE_BUCKET_NAME": os.environ.get(
+        "AWS_STORAGE_BUCKET_NAME", "evalai-s3-bucket"
+    ),
+}
+
+COMMON_SETTINGS_DICT = {
+    "EXECUTION_ROLE_ARN": os.environ.get(
+        "EXECUTION_ROLE_ARN",
+        "arn:aws:iam::{}:role/evalaiTaskExecutionRole".format(
+            aws_keys["AWS_ACCOUNT_ID"]
+        ),
+    ),
+    "WORKER_IMAGE": os.environ.get(
+        "WORKER_IMAGE",
+        "{}.dkr.ecr.us-east-1.amazonaws.com/evalai-{}-worker:latest".format(
+            aws_keys["AWS_ACCOUNT_ID"], ENV
+        ),
+    ),
+    "CODE_UPLOAD_WORKER_IMAGE": os.environ.get(
+        "CODE_UPLOAD_WORKER_IMAGE",
+        "{}.dkr.ecr.us-east-1.amazonaws.com/evalai-{}-worker:latest".format(
+            aws_keys["AWS_ACCOUNT_ID"], ENV
+        ),
+    ),
+    "CIDR": os.environ.get("CIDR"),
+    "CLUSTER": os.environ.get("CLUSTER", "evalai-prod-cluster"),
+    "DJANGO_SERVER": os.environ.get("DJANGO_SERVER", "localhost"),
+    "EVALAI_API_SERVER": os.environ.get("EVALAI_API_SERVER", "localhost"),
+    "DEBUG": settings.DEBUG,
+    "EMAIL_HOST": settings.EMAIL_HOST,
+    "EMAIL_HOST_PASSWORD": settings.EMAIL_HOST_PASSWORD,
+    "EMAIL_HOST_USER": settings.EMAIL_HOST_USER,
+    "EMAIL_PORT": settings.EMAIL_PORT,
+    "EMAIL_USE_TLS": settings.EMAIL_USE_TLS,
+    "MEMCACHED_LOCATION": os.environ.get("MEMCACHED_LOCATION", None),
+    "RDS_DB_NAME": settings.DATABASES["default"]["NAME"],
+    "RDS_HOSTNAME": settings.DATABASES["default"]["HOST"],
+    "RDS_PASSWORD": settings.DATABASES["default"]["PASSWORD"],
+    "RDS_USERNAME": settings.DATABASES["default"]["USER"],
+    "RDS_PORT": settings.DATABASES["default"]["PORT"],
+    "SECRET_KEY": settings.SECRET_KEY,
+    "SENTRY_URL": os.environ.get("SENTRY_URL"),
+    "STATSD_ENDPOINT": os.environ.get("STATSD_ENDPOINT"),
+    "STATSD_PORT": os.environ.get("STATSD_PORT"),
```

Comment:
Are all of these used?

Reply:
No, only the CLUSTER one. I'll remove the unused